guides/v2.3/graphql/reference/categories.md

```diff
@@ -4,11 +4,11 @@ title: category endpoint
 version: 2.3
 ---
 
-The `category` endpoint allows you to search for a single category definition or the entire category tree.
+The `category` endpoint allows you to search for a single category definition or the entire category tree. To return multiple category levels in a single call, define the response so that it contains up to ten nested `children` options. You cannot return the entire category tree if it contains more than 10 sublevels.
 
```

Comment:
Not sure if it helps, but consider adding a "for example" 

define a response so that it contains up to ten nested `children` options, for example the query can return a CategoryTree object with up to ten levels of categories, and each of those categories can contain up to ten CategoryTree objects with up to ten subcategories. 


Also, in the Sample Queries section --  It might be good to show a real response for the CategoryTree query since it's kind of complicated.

Reply:
I left the description as is, but added sample output.